### ISSUE TITLE
Extend Move operation to work between Arrays and Objects

### ISSUE THREAD

**Issue by hackerwins** [yorkie-team/yorkie#83]
The move operation still works only on the elements of an Array. It would be nice to expand this in the future so that it can be moved between Objects and Arrays.

https://github.com/yorkie-team/yorkie/issues/43#issuecomment-612295610

We have an example of Kanban board on the demo page, and by completing this function we can show an example of moving cards between boards.

https://yorkie.dev/demo

**Comment by jeemyeong** [comment]
To do this, I think refactoring `move operation` to support both `ParentCreatedAt` and `PrevParentCreatedAt` is one of solutions. 
How do you think?
If it is correct, I will try it (including yorkie-js-sdk)

**Comment by hackerwins** [comment]
@jeemyeong 

That's right. Since the target element needs to be moved between different parent elements, we need to pass `PrevParentCreatedAt` which is the previous parent's position and `ParentCreatedAt` which is the current parent's position to MoveOperation.
